Driver_FET.lib

```diff
@@ -1544,6 +1602,35 @@ X LO 9 300 -300 100 L 50 50 1 1 O
 ENDDRAW
 ENDDEF
 #
+# L6491
+#
+DEF L6491 U 0 20 Y Y 1 F N
+F0 "U" 50 525 50 H V L CNN
+F1 "L6491" 50 450 50 H V L CNN
+F2 "Package_SO:SOIC-14_3.9x8.7mm_P1.27mm" 0 950 50 H I C CIN
+F3 "" 0 0 50 H I C CNN
+$FPLIST
+ SOIC*3.9x8.7mm*P1.27mm*
+$ENDFPLIST
+DRAW
+S -300 -400 300 400 0 1 10 f
+X ~LIN 1 -400 0 100 R 50 50 1 1 I
+X CP+ 10 400 -200 100 L 50 50 1 1 P
+X NC 11 -300 300 100 R 50 50 1 1 N N
+X OUT 12 400 100 100 L 50 50 1 1 P
+X HVG 13 400 200 100 L 50 50 1 1 O
+X BOOT 14 400 300 100 L 50 50 1 1 P
+X ~SD/~OD 2 -400 -100 100 R 50 50 1 1 C
```

Comment:
The second `~` should be to the left of the `/`.

Reply:
Done